Protocol.md

```diff
@@ -309,8 +309,9 @@ The Error Data is typically an Exception message, but could include stringified
 | __INVALID_SETUP__              | 0x00000001 | The Setup frame is invalid for the server (it could be that the client is too recent for the old server). Stream ID MUST be 0. |
 | __UNSUPPORTED_SETUP__          | 0x00000002 | Some (or all) of the parameters specified by the client are unsupported by the server. Stream ID MUST be 0. |
 | __REJECTED_SETUP__             | 0x00000003 | The server rejected the setup, it can specify the reason in the payload. Stream ID MUST be 0. |
-| __CONNECTION_ERROR__           | 0x00000101 | The connection is being terminated. Stream ID MUST be 0. |
+| __CONNECTION_ERROR__           | 0x00000101 | The connection is being terminated. Stream ID MUST be 0. Sender or Receiver of this frame MAY not wait for outstanding streams to terminate before closing the connection.|
```

Comment:
The "MAY not wait" part required me re-reading a couple times as it comes across as a double negative for me. Perhaps clearer worded as follows?

> Sender or Receiver of this frame MAY close the connection immediately without waiting for outstanding streams to terminate.